gulpfile.js

```diff
@@ -1,8 +1,10 @@
 var gulp = require('gulp');
+var webserver = require('gulp-webserver');
 var browserify = require('browserify');
 var source = require('vinyl-source-stream');
 var tsify = require('tsify');
 var sass = require('gulp-sass');
+var open = require('gulp-open');
 var paths = {
     pages: ['./public/*.html', './public/**/images/**']
 };
```

Comment:
You should've created another function for webserver:

```javascript
function liveReload() {
    gulp.src('dist/').pipe(webserver({
        livereload: true,
        open: "index.html"
    }));
}
```

Reply:
Ok, seems good. I was not really sure how to fit it in the exports assignments, but as you showed later, parallel works well